packages/jest-editor-support/src/TestReconciler.js

```diff
@@ -108,15 +108,24 @@ module.exports = class TestReconciler {
 
   // Do everything we can to try make a one-liner from the error report
   sanitizeShortErrorMessage(string: string): string {
+    if (string.includes("does not match stored snapshot")) {
```

Comment:
I had a look at this, but the way things are pieced together is like this:
 
![screen shot 2017-05-19 at 16 24 28](https://cloud.githubusercontent.com/assets/49038/26254819/bbea241a-3caf-11e7-97ff-efc7d3fdb536.png)

There is a [this test](https://github.com/facebook/jest/blob/master/integration_tests/__tests__/toMatchSnapshot-test.js#L41)  which incidentally covers this case, I could add comments to?